python-package/lightgbm/dask.py

```diff
@@ -384,6 +385,9 @@ def _predict(model, data, raw_score=False, pred_proba=False, pred_leaf=False, pr
 
 
 class _LGBMModel:
+    def __init__(self):
+        if not all((DASK_INSTALLED, PANDAS_INSTALLED, SKLEARN_INSTALLED)):
+            raise LightGBMError('Dask, Pandas and Scikit-learn are required for this module')
```

Comment:
```suggestion
            raise LightGBMError('dask, pandas and scikit-learn are required for lightgbm.dask')
```

Instead of "this module", could you use the specific name? I think that makes the log message a little more useful standalone. It can be helpful for cases where people don't have direct access to the stack trace, which is required to understand what "this module" refers to.

For example, user  code or other frameworks might write things like this

```python
try:
    dask_reg = DaskLGBMClassifier()
except LightGBMError as err:
    log.fatal(err)
    raise SomeOtherException("LightGBM training failed")
```

I also think packages should be referenced by their exact package names, not capitalized names.

Reply:
Thank you! Addressed in acac78fe852cef265a9dda4e4c0c3ddf30b91a72.